ibis/pandas/execution/window.py

```diff
@@ -264,7 +267,11 @@ def execute_window_op(
                 grouping_keys,
                 ordering_keys,
             ) = util.compute_sorted_frame(
-                data, order_by, group_by=group_by, **kwargs
+                data,
+                order_by,
+                group_by=group_by,
+                timecontext=new_timecontext,
```

Comment:
Can we rename `new_timecontext` to `adjusted_timecontext`?

Reply:
Sure